tsconfig-strict.json

```diff
@@ -184,8 +184,6 @@
     "core/templates/domain/learner_dashboard/learner-dashboard-activity-ids.model.spec.ts",
     "core/templates/domain/learner_dashboard/learner-dashboard-ids-backend-api.service.ts",
     "core/templates/domain/learner_dashboard/learner-dashboard-ids-backend-api.service.spec.ts",
-    "core/templates/domain/learner_dashboard/non-existent-activities.model.ts",
-    "core/templates/domain/learner_dashboard/non-existent-activities.model.spec.ts",
```

Comment:
While you remove the old files from here can you introduce the newly added files here? So that we don't lose strict typing coverage?